Revert all incompatible changes

joshxg opened the pull request:
#454 

Well I hope this isn't a bad idea. 

 I think this will be all round better as we can implement the more complex things like handling errors

all up this took about 30 minutes. Should only take less to revert things as they come in to the server

mark-prins commented on packages/host/public/config.js:
```diff
@@ -1,4 +1,4 @@
-  API_URL: 'http://localhost:4000',
-  // API_URL: 'http://localhost:8000/graphql',
+  // API_URL: 'http://localhost:4000',
+  API_URL: 'http://localhost:8000/graphql',
```
I can update the version on demo-open.msupply.org and we could commit that URL rather than requiring a locally running server

mark-prins commented on packages/host/src/bootstrap.tsx:
```diff
@@ -9,7 +9,7 @@ if (process.env.NODE_ENV === 'development') {
   const worker = setupMockWorker();
 
-  worker.start();
+  // worker.start();
```
would the msw be permanently disabled?

mark-prins reviewed (approved):
😢  bye bye all you beautiful unsupported things. I'll see you again one day..

one final thing - I'm getting an error when trying to add items - might be just me? I think it could be the dates in my stock lines.. just check that it's ok for you?
```
"availableBatches": {
                        "__typename": "ConnectorError",
                        "error": {
                            "description": "Database Error",
                            "__typename": "DatabaseError",
                            "fullError": "\"DIESEL_DESERIALIZATION_ERROR\" (\"ParseError(Invalid)\")"
                        }
                    },
```

joshxg commented:
Changed the URL and reverted the disabling of the msw. Also, not getting an error for adding 🤔  Could very well be an issue that is on my end, not yours!Will move forward for now though
